application.go

```diff
@@ -241,6 +247,13 @@ EventLoop:
 				a.RLock()
 				screen := a.screen
 				a.RUnlock()
+
+				// Call afterResize handler if there is one.
+				if a.afterResize != nil {
+					width, height := screen.Size()
```

Comment:
This will panic if `screen` is `nil` (which can happen, as can be seen further below). Can you move this block to after the `screen == nil` check?

Reply:
Thanks. PR updated.